README.md

```diff
@@ -28,10 +28,7 @@ If you'd like to contribute, please review the
 ## Overview
 
 This repository includes installation resources to deploy the full Zenko
-stack over different orchestration systems.
-
-Currently we have [Kubernetes](https://kubernetes.io/) and
-[Docker Swarm](https://docs.docker.com/engine/swarm/).
+stack on the [Kubernetes](https://kubernetes.io/)  orchestration system.
```

Comment:
```suggestion
stack on the [Kubernetes](https://kubernetes.io/) orchestration system.
```
There is a double space if you care to delete it, it doesn't seem to affect the rendered output though.

Reply:
Done, though if CI doesn't catch it I should've ignored it :wink:

Reply:
I don't think I have ever seen any linting catch double spaces, it is just something I notice in reviews.